MAINTAINERS.yml

```diff
@@ -2831,9 +2844,9 @@ x86 arch:
 Continuous Integration:
   status: maintained
   maintainers:
-    - stephanosio
     - nashif
     - galak
+    - stephanosio
   files:
```

Comment:
@gramsay0 nitpick, I think you should rebase to not change the order of maintainers here, see https://github.com/zephyrproject-rtos/zephyr/commit/f05229c6bc7fed9cdbf670f9e16a94e98f945543

Reply:
Fixed